lib/driver.rb

```diff
@@ -43,5 +44,67 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def get_total_revenue
+      total_rev = 0.0
+      @trips.each do |trip|
+        if trip.cost != nil
+          trip_cost = (trip.cost - 1.65) * 0.8
+          total_rev += trip_cost.to_f
+        end
+      end
+      return total_rev.round(2)
```

Comment:
You don't need an explicit `return` here, you can remove the word `return`. Idiomatic Ruby doesn't usually use the `return` keyword unless you need to return a value before the end of the method.